docs/src/examples/01-loading-and-saving.jl

```diff
@@ -38,9 +38,9 @@ download_model(
     "e_coli_core.xml",
     "b4db506aeed0e434c1f5f1fdd35feda0dfe5d82badcfda0e9d1342335ab31116",
 )
-
-# (You do not need to fill in the hash values immediately; simply run the
-# function once and then copy the reported hash value into your script.)
+#!!! tip "Tip: How do I get the model hash?"
+#    You do not need to fill in the hash values immediately; simply run the
+#    function once, and then copy the reported hash value into your script.
```

Comment:
```suggestion
#md # !!! tip "Tip: How do I get the model hash?"
#md #    You do not need to fill in the hash values immediately; simply run the function once, and then copy the reported hash value into your script.
```

(follow the style from e.g. https://github.com/COBREXA/COBREXA.jl/blob/68d173dd566a0b523a4f17294f2e57c0a773aff8/docs/src/examples/02b-model-modifications.jl#L140 )